cln-grpc/src/pb.rs

```diff
@@ -31,7 +31,7 @@ impl From<JOutpoint> for Outpoint {
 impl From<Outpoint> for JOutpoint {
     fn from(a: Outpoint) -> Self {
         JOutpoint {
-            txid: bitcoin_hashes::sha256::Hash::from_slice(&a.txid).unwrap(),
+            txid: bitcoin::hashes::sha256::Hash::from_slice(&a.txid).unwrap(),
```

Comment:
```suggestion
            txid: Hash::from_slice(&a.txid).unwrap(),
```

Reply:
I'm afraid not: `bitcoin::hashes::Hash` is a trait, whereas `bitcoin::hashes::sha256::Hash` is a struct implementing that trait.

Reply:
Oh man, this is confusing, yeah I got the point now!